lib/shared/bucketing/src/bucketing.ts

```diff
@@ -1,5 +1,8 @@
 'use strict'
-import { orderBy, pick, last, first } from 'lodash'
```

Comment:
can you add this lint rule to prevent this in the future?
https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/import-scope.md

Reply:
Oh good call

Reply:
Added